### DISCUSSION TITLE
Installazione del registro su Aruba

### DISCUSSION THREAD

**Discussion by ghost** [iisgiua/giuaschool#628] [Richieste di aiuto] [answered]
Buongiorno @trinko, come posso installare il registro su database MySql o MsSql di Aruba?
Ci sono delle istruzioni specifiche, oppure devo usare sempre le solite?
Chiedo aiuto anche a @manuelbarbieri @bfabio @zerai @MatteoGheza.

Grazie

**Comment by trinko** [comment]
Non ci sono istruzioni specifiche. I requisiti fondamentali sono:
- uso di MySQL 5.5 o superiore, oppure MariaDB 10.0 o superiore;
- uso PHP 7.4 o superiore.

Per il resto segui la documentazione: [Installazione tramite archivio ZIP](https://iisgiua.github.io/giuaschool-docs/install-zip.html)

**Comment by ghost** [comment]
Buongiorno Antonello, ho un problema.
Ok, tramite Hosting Linux su Aruba (collegato a database MySql) sonoriuscito ad accedere alla pagina di installazione, ma c'è un problema. Qui nell'immagine, cosa dovrei mettere?

**Reply by ghost** [reply]
![image](https://github.com/iisgiua/giuaschool/assets/146358960/23e3c05b-0c30-4ad0-b27c-608868c013b7)

**Reply by ghost** [reply, marked as answer]
Risolto.
